### DISCUSSION TITLE
I need help with a last stand mechanic

### DISCUSSION THREAD

**Discussion by mcdjhorton** [Pryaxis/TShock#1724] [General]
Last Stand mechanic is in my server, if everyone in the server gets put into the green team but there's one survivor left, the survivor will get multiple buffs for 2 minutes to aid the survivor from all the green team zombies from killing them.

**Comment by moisterrific** [comment]
Is this a plugin?

**Reply by mcdjhorton** [reply]
yeah it's supposed to be but with other plugins

**Comment by hakusaro** [comment]
Is this just #1720?
